model/code/registry-deprecated.yaml

```diff
@@ -8,35 +8,48 @@ groups:
       - id: code.function
         type: string
         stability: development
-        deprecated: Replaced by `code.function.name`
+        deprecated:
+          reason: uncategorized
+          note: >
```

Comment:
Why don't we use the `renamed_to` here? 🤔

Reply:
because `code.function` is not qualified e.g. `foo` and  `code.function.name` is fully qualified, e.g. `my_app.namespace.class.foo`, so in order to construct `code.function.name` you'd need to take deprecated  `code.namespace` and `code.function`.

`renamed_to` is essentially limited to renames that don't change anything important about the attribute (type, meaning, format).

we could in the future define more complex transformations, it's just not supported now.

Hope it helps!